tests/models/decoder_only/vision_language/utils.py

```diff
@@ -0,0 +1,999 @@
+"""Common utility functions relating to different models that are useful
```

Comment:
This file is pretty long. Can we split out each part into a separate file?

Reply:
Agreed! I'll split it up 😄

Reply:
I broke it up into a `vlm_utils` package. For now, it's still in the VLM tests, but I think we can also pull it up and bring other things into it in follow-ups if we want to (e.g., `audio` tests which be similar to the same changes I made for `video`, or the encoder/decoder vision tests for mllama)